src/Quizzer.API.BL/ApiBLInstaller.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Quizzer.API.BL.Facades;
+using Quizzer.API.DAL.UnitOfWork;
+using Quizzer.Extensions;
+
+namespace Quizzer.API.BL {
+    public class ApiBLInstaller : IDependencyInstaller {
+        public void Install(IServiceCollection services) {
+            services.AddSingleton<IUnitOfWorkFactory, UnitOfWorkFactory>();
+
+            services.Scan(selector => selector
+            .FromAssembliesOf(typeof(IFacade<,,>))
```

Comment:
In cookbook there is `selector.FromAssemblyOf<ApiBLInstaller>()`. Can you explain why we do this different?

Reply:
This code snippet is taken from ActiveTracker and which class is used to define the searching assembly is not important as long as it is contained in the same assembly as dependencies we want to register. And since I think the best class for scoping the assembly is the dependency itself, I left the Facade to be defining the assembly scope. But for the sake of completion your complaint, I think The BL installer class would work the same.